rules/windows/builtin/win_invoke_obfuscation_var+_services.yml

```diff
@@ -0,0 +1,43 @@
+action: global
+title: Invoke-Obfuscation VAR+ Launcher
+id: 8ca7004b-e620-4ecb-870e-86129b5b8e75
+description: Detects Obfuscated use of Environment Variables to execute PowerShell
+status: experimental
+author: Jonathan Cheong, oscd.community
+date: 2020/10/15
+references:
+     - https://github.com/Neo23x0/sigma/issues/1009 #(Task 24)
+tags:
+    - attack.defense_evasion
+    - attack.t1027
+    - attack.execution
+    - attack.t1059.001
+falsepositives:
+    - Unknown
+level: high
+detection:
+    selection_1:
+        - ImagePath|re: 'set\s[a-zA-Z]{3,6}=Invoke-Expression'
+        - ImagePath|re: '(\"(?:\{\d\}){1,7}\\){1,5}'
+    condition: selection and selection_1
```

Comment:
Hey @OpalSec, nice contribution, but this condition references `selection` before it gets declared